hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/DroppedFrameObserver.java

```diff
@@ -0,0 +1,109 @@
+package com.willowtreeapps.hyperion.geigercounter;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.res.AssetFileDescriptor;
+import android.media.SoundPool;
+import android.util.Log;
+import android.view.Choreographer;
+
+import static android.media.AudioManager.STREAM_SYSTEM;
+import static com.willowtreeapps.hyperion.geigercounter.GeigerCounterPlugin.LOG_TAG;
+
+@SuppressLint("NewApi")
```

Comment:
I think it'd be better to `TargetApi()` this to make it more clear which api this would work in.